ex02.exs

```diff
@@ -1,6 +1,28 @@
 
-defmodule Ex02 do
 
+defmodule Ex02 do
+  @global_count {:global, __MODULE__}
+
+  def new_counter(initial_value \\ 0) do
+    { :ok, pid } = Agent.start_link(fn -> initial_value end)
+    pid
+  end
+
```

Comment:
There's a race condition in the `next_value` function which could lead to two calls from different processes both getting the same value. Can you see where?

# -3